js/components/Color.js

```diff
@@ -0,0 +1,11 @@
+const React = require("react");
+
+class Color extends React.Component {
+	render() {
+		return (
+			<p className="app-color" font size = "3" color= "blue">Hello There!</p>
```

Comment:
The attributes here (Size and color) should not be hard-coded, but instead by taken in as props.